deploy/install/01-prerequisite/03-crd.yaml

```diff
@@ -113,6 +201,22 @@ spec:
     singular: node
   scope: Namespaced
   version: v1beta1
+  additionalPrinterColumns:
+  - name: Ready
+    type: string
+    description: Indicate whether the node is ready
+    JSONPath: .status.conditions['Ready']['status']
+  - name: AllowReplicaScheduling
```

Comment:
`AllowScheduling` is enough.

Reply:
Thank you! Fixed

Reply:
well, I think it's a misunderstanding.

I meant rename the field to `AllowScheduling`, not `AllowReplicaScheduling`.

We can keep the `scheduable` condition.

Reply:
Other than this, LGTM.

Reply:
Sorry. I misunderstand it. Fixed